testdata/expected/docker/java-android-gradle-sample/default-group/java-android-gradle-sample/JavaArtifact.graph.json

```diff
@@ -1,1241 +0,0 @@
-{
```

Comment:
Why does the Docker version miss all these defs and refs now? A lot of them do *not* seem to be in generated code, and therefore should be detected, right?